docs/project/changelog.md

```diff
@@ -26,6 +26,10 @@ myst:
   functions.
   {pr}`4392`
 
+- {{ Breaking }} Pyodide will not fallback to `node-fetch` anymore when `fetch`
+  is not available in the Node.js environment.
```

Comment:
Is this really breaking since we already said we don't support Node 16? Maybe mention Node 16 in the message too?

Reply:
Thanks. I updated the changelog that it affects Node.js < 18.